### PR TITLE
Created Drop in Field for File Upload

### PR DESCRIPTION
**Pull request by MaxTyrchan**

### CODE REVIEW

File: src/pages/UploadPage.tsx [hunk 1]
```diff
@@ -58,59 +64,14 @@ export default function UploadPage() {
...
-              if (!event.target.files) return;
-              setCoverLetter(event.target.files[0]);
-            }}
-          />
-        </Button>
-      </Box>
       <Box sx={{ display: 'flex', flexDirection: 'row', mb: 2 }}>
-        <FormControl sx={{ width: 300 }} size="small">
-          <InputLabel id="vacancy-label">Vacancy</InputLabel>
-          <Select
```
Comment (ScottPalme) [inline src/pages/UploadPage.tsx:100]: Please do not delete the drop down. The endpoint need both the CV and the selected vacancy to rate the applicant

File: src/pages/UploadPage.tsx [hunk 2]
```diff
@@ -34,15 +35,20 @@ export default function UploadPage() {
     fetchVacancies();
   }, []);
 
+  const setFile = (file: File) => {
+    setCV(file);
+    setSuccess(true);
```
Comment (ScottPalme) [inline src/pages/UploadPage.tsx:40]: This should not be set to true since it has not been send after being set

File: src/pages/UploadPage.tsx [hunk 3]
```diff
@@ -58,59 +64,14 @@ export default function UploadPage() {
...
-              <MenuItem key={vacancy.id} value={vacancy.id} sx={{ height: '35px' }}>
-                {vacancy.title}
-              </MenuItem>
-            ))}
-          </Select>
-        </FormControl>
+        <DropZone setFile={setFile} />
       </Box>
       <Box sx={{ display: 'flex', flexDirection: 'row', mb: 30 }}>
         <Button
```
Comment (ScottPalme) [inline src/pages/UploadPage.tsx:77]: This button stays disabled after dropping the file

**Review by ScottPalme — CHANGES_REQUESTED** [review]
There are some things that need to be changed

File: src/pages/UploadPage.tsx (hunk 3 above)
Comment (MaxTyrchan) [reply src/pages/UploadPage.tsx:77]: That is because no vacancy is selected (see line 66)

**Review by ScottPalme — APPROVED** [review]

**Pull request merged**